lessons/module-3/react-2-the-how.md

```diff
@@ -69,18 +70,7 @@ Installing react, react-dom, and react-scripts...
 When the script is finished running, you will see the following message in your terminal:
 
 ```bash
-> node scripts/postinstall || echo "ignore"
-
-+ react-dom@16.8.6
-+ react@16.8.6
-+ react-scripts@3.0.1
-added 1541 packages from 747 contributors and audited 888986 packages in 52.528s
-found 0 vulnerabilities
-
-
-Initialized a git repository.
-
-Success! Created ideabox at /Users/leta/Turing/TA/MOD3/m3curriculum/ideabox
+Success! Created ideabox at /Users/yourname/Turing/TA/MOD3/m3curriculum/ideabox
```

Comment:
Thanks for updating this!

Reply:
Since you updated this here to `yourname`, I would also update this on line 62 as well where is says `Creating a new React app in /Users/leta/mod3/ideabox.`